pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -118,6 +118,13 @@
 ,
 }: self: super:
 {
+  agitator-nvim = super.agitator-nvim.overrideAttrs {
```

Comment:
This file is alphabetically sorted.

Reply:
Also says there is a conflict. I think you are basing this on an outdated master branch.

Reply:
Yeah I used `origin/master` instead of `uplstream/master` :(

File seems to be a little out of order, but I've done my best.